static/js/settings.js

```diff
@@ -45,16 +45,16 @@ function setup_settings_label() {
     settings_label = {
         // settings_notification
         presence_enabled: $t({
-            defaultMessage: "Display my availability to other users when online",
+            defaultMessage: "Display my availability to other users. (invisible mode off)",
         }),
         send_stream_typing_notifications: $t({
-            defaultMessage: "Let subscribers see when I'm typing messages in streams",
+            defaultMessage: "Let subscribers see when I'm typing messages in streams.",
         }),
         send_private_typing_notifications: $t({
-            defaultMessage: "Let recipients see when I'm typing private messages",
+            defaultMessage: "Let recipients see when I'm typing private messages.",
```

Comment:
None of our other settings use `.s` at the end of their labels; so I undid these changes.

Reply:
Also added a commit to remove the `.` from the one recently added setting that has one.